src/com/comeet/UserService.java

```diff
@@ -120,6 +120,47 @@
 
 
     /**
+     * Implementing method for GET /{organization}/meetings/attendees?id=
+     * <p>
+     * Example URL:
+     * </p>
+     * 
+     * @param orgDomain Start of query range.
+     * @param id - meeting ID
+     * @return Meeting info within the query range.
+     * @throws ServiceResponseException If the result is not 200 OK
+     * @throws Exception On an unexpected error.
+     */
+    @POST
+    @Path("/{orgDomain}/meetings/attendees")
+    @Produces("application/json")
+    @Consumes(MediaType.APPLICATION_FORM_URLENCODED)
+    public Attendees getAttendees(@Context HttpHeaders headers,
+                    @PathParam("orgDomain") String orgDomain, 
+                    @DefaultValue("") @FormParam("id") String id)
```

Comment:
Meeting needs to expose the id.
Please fork the api builder spec with these changes.
https://github.com/bookaroom/dev-docs/blob/master/apiary.apib